core/src/main/java/org/everit/json/schema/NumberSchemaValidatingVisitor.java

```diff
@@ -3,9 +3,18 @@
 import static java.lang.String.format;
 
 import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.util.HashSet;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicLong;
+
+import com.google.common.collect.Sets;
```

Comment:
Currently we don't use guava, this import doesn't compile on CI ( https://travis-ci.org/everit-org/json-schema/builds/404507721 )

Reply:
right